AppBidding-MoPub/AppBidding-MoPub/BannerViewController.m

```diff
@@ -0,0 +1,43 @@
+//
+//  BannerViewController.m
+//  AppBidding-MoPub
+//
+//  Created by Abdoul Guisset on 6/29/20.
+//  Copyright © 2020 Abdoul. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "BannerViewController.h"
+#import "AdConfigurations.h"
+@import CriteoPublisherSdk;
+
+@interface BannerViewController ()
+
+@end
+
+@implementation BannerViewController
+
+
+// safeAreaInsets will be 0 if executed in viewDidLoad
```

Comment:
What do you mean by this ? Could you elaborate this comment ?

Reply:
The safeArea is only available through the viewDidAppear method. If we decide to go with viewDidLoad, the safe area would be set to 0. I can add this as a comment if needed.

Reply:
Sure, I think that it would be great to have such comment in the code to explain the choice of viewDidAppear rather than viewDidLoad.